### PR TITLE
fix(webserver): fix repositories created from Github / Gitlab provide…

### PR DESCRIPTION
**Pull request by wsxiaoys**
…rs being wrongly garbage collected

### CODE REVIEW

File: ee/tabby-webserver/src/service/background_job/git.rs [hunk 1]
```diff
@@ -52,10 +52,6 @@ impl SchedulerGitJob {
             .map(|repo| CodeRepository::new(&repo.git_url, &repo.source_id))
             .collect();
 
-        let mut code = CodeIndexer::default();
-
-        code.garbage_collection(&repositories).await;
```
Comment (wsxiaoys) [inline ee/tabby-webserver/src/service/background_job/git.rs:57]: only contains repositories from git provider, thus the cloned dirs from github / gitlab will be wrongly deleted.

@xxs-wallace @liangfung

**Review by xxs-wallace — APPROVED** [review]
LGTM

File: ee/tabby-webserver/src/service/background_job/git.rs (hunk 1 above)
Comment (xxs-wallace) [reply ee/tabby-webserver/src/service/background_job/git.rs:57]: Can we add repository from github/gitlab to this variable `repositories` ?

Comment (wsxiaoys) [reply ee/tabby-webserver/src/service/background_job/git.rs:57]: > Can we add repository from github/gitlab to this variable repositories ?

it's done in IndexGarbageCollection now

**Pull request merged**